Sources/StreamChatUI/Gallery/GalleryVC.swift

```diff
@@ -341,8 +341,10 @@ open class GalleryVC:
             withReuseIdentifier: reuseIdentifier,
             for: indexPath
         ) as! GalleryCollectionViewCell
-        
-        cell.content = items[indexPath.item]
+
+        guard let item = getItem(at: indexPath) else { return UICollectionViewCell() }
```

Comment:
Lets assert this place, too.

Reply:
I wrapped the logging & assertion logic in getItem